src/utils/useTrackTopic.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { getTrackingEnv } from "./trackingUtils";
+
+let prevUrl;
+
+function useTrackTopic(topicKey) {
+  const { i18n } = useTranslation();
+  useEffect(() => {
+    const env = getTrackingEnv();
+    if (env && topicKey) {
+      window.digitalDataLayer = window.digitalDataLayer || [];
+      window.digitalDataLayer.push({
+        pageInstanceID: env === "prod" ? "611081" : "611157",
```

Comment:
Damit klar ist, das hier noch was angepasst werden muss:
```suggestion
        pageInstanceID: env === "prod" ? "TODO: prod pageInstanceID" : "TODO: stag pageInstanceID",
```